### ISSUE TITLE
can the dashboard rotate builds in the view if many projects are configured

### ISSUE THREAD

**Issue by jmecosta** [AdamNowotny/BuildReactor#47]
hi,

seems i cannot get the dashboard to show all builds in screen if there are 2 many builds. is this something that can be done?

thanks and good job with this.

br,
jorge costa

**Comment by AdamNowotny** [comment]
You can zoom in and out to make builds take all available space (ctrl-+, ctrl--)

**Comment by jmecosta** [comment]
well, that means no one will be able to read the commit authors :) and we use chrome and change tabs every few seconds so the zoom will aftect negatively everything else. so we can use that approach, any more ideas?

**Comment by AdamNowotny** [comment]
How about grouping successful builds ("x builds passed" block) if amount of builds in a group exceeds configured amount or does not fit in one row?
Someone requested a while ago filtering to only failed builds, would that be useful?

**Comment by jmecosta** [comment]
yes that can work, but i am not sure how the groping will be done when you have different branches see picture
![image](https://cloud.githubusercontent.com/assets/1041352/14891531/bbd2e784-0d6f-11e6-95fa-40cf15847fbf.png)

also from picture there is loads of empty space taken by project names that i think could be just removed

**Comment by Iriani** [comment]
Would it be possible to configure multiple dashboards on 1 machine?
Using the revolver Tabs chrome plugin, I could then display them all on 1 monitor

**Comment by AdamNowotny** [comment]
With version 4.0.0, you can create your own visualisations using React, this would likely involve creating your own theme that uses a modified Pipelines component.
